judo-ui-react/src/main/resources/actor/src/fragments/relation/hook-variables.fragment.hbs

```diff
@@ -1,9 +1,10 @@
 const {{ rel.dataElement.name }}Columns: GridColDef<{{ classDataName rel.dataElement.target 'Stored' }}>[] = [
     {{# each rel.columns as |column| }}
+        {{ debug column }}
```

Comment:
This debug can only be found here. Is it intentional?

Reply:
nope